README.md

```diff
@@ -4,7 +4,54 @@
 
 ## Overview
 
-*To be written*
+Agentmon reads language metrics from Prometheus or a StatsD source and posts them to the language metrics sink, for 
+example `https://app.metrics.heroku.com/<dyno id>`.
+```bash
+usage: agentmon [flags] sink-URL 
+ -backlog int
+        Size of pending measurement buffer (default 1000)
+  -debug
+        debug mode is more verbose
+  -interval int
+        Sink flush interval in seconds (default 20)
+  -prom-interval int
+        Prometheus poll interval in seconds (default 5)
+  -prom-url string
+        Prometheus URL
+  -statsd-addr string
+        UDP port for statsd listener
+  -version
+        print version string
+```
+Language metrics are supported for Go, JVM, and Ruby. 
+
+#### Go Metrics 
```

Comment:
It really doesn't make sense to document go metrics, or other language metrics here. It acts as an intermediary.

Reply:
Therefore, should probably not be in here.